docs/swagger.yaml

```diff
@@ -337,7 +337,7 @@ components:
       required: true
       schema:
         type: string
-      description: Requested image format
+      description: 'Requested image format. Supported values: {image/*,image/png, image/svg+xml, text/targetUrl} '
```

Comment:
Can this be made to a required enum?

Reply:
I tried with enums, but values like '/' and '*' is not supported. 
Could change to only accept 
png, svg and targetUrl